src/Microsoft.Azure.NotificationHubs/RetryPolicy.cs

```diff
@@ -0,0 +1,30 @@
+﻿//------------------------------------------------------------
```

Comment:
We don't need a separate class for RetryPolicy, lets copy the default policy to NamespaceSettings class. The the default ctor() we should assign the default policy.